src/main/webapp/app/overview/exercise-details/header-exercise-page-with-details.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="course-info-bar" *ngIf="exercise">
```

Comment:
I like the idea about a component for the header, but i think the overview.module is the wrong place for it, because if you just wan to add this one header component you need to add the whole overview module. maybe an own module would be better here